js/main.js

```diff
@@ -48,6 +48,15 @@ jQuery(document).on('ready', function() {
 			scrollTop: jQuery("#tg-main").offset().top + offset
 		}, 2000);
 	});
+
+	/* -------------------------------------
+	     ADD ANIMATIONS TO SECTION SCROLL
+	 -------------------------------------- */
+    $('a[href*=\\#]').on('click', function(event){
+        event.preventDefault();
```

Comment:
This makes 'Speakers' button a dead link when clicked from any other page than index.html. Different approach is needed to make it work.